samples/e2eTestUtils/src/Deserializer.ts

```diff
@@ -1,191 +0,0 @@
-/*
```

Comment:
This class as well as Serializer in the test utils are unecessary, they can be imported from msal-node directly.